File: Java/Answer5.java
class  Answer5 {
    public static void main(String[] args) {
        int c = 0,a, temp;
        int n=153;
        temp = n;
        System.out.print(n+" is ");
        while (n>0) {
            a = n%10;
            n = n/10;
            c = c+(a*a*a);
        }
        if(temp == c){
            System.out.println("an arm strong number");
        } else {
            System.out.println("not an armstrong number");
        }

    }
}

File: C++/Answer5.cpp
#include<iostream>
using namespace std;

int main () {
    int c = 0, a, temp;
    int n = 153;
    temp = n;
    std::cout << n << " is ";
    while (n>0) {
        a = n%10;
        n = n/10;
        c = c + (a*a*a);
    }
    if (temp == c) {
        std::cout << "an armstrong number" << std::endl;
    } else {
        std::cout << "not an armstrong number" << std::endl;
    }
    return 0;
}